src/diart/models.py

```diff
@@ -11,12 +11,34 @@
     from pyannote.audio.pipelines.speaker_verification import (
```

Comment:
I can't comment the line above but `Inference` isn't used, this one escaped us in #188. Could you change it to:

```python
from pyannote.audio import Model
```